README.md

```diff
@@ -106,6 +106,13 @@ There are also some basic E2E tests, they are prepared to run using
 make e2e-test
 ```
 
+The e2e tests make the assumption that you are running on an AMD system so in case the test doesn't generate the needed binary, run the below command. 
```

Comment:
I cannot tell if [the lines](https://github.com/newrelic/k8s-metadata-injection/pull/311/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R76-R87) about `skaffold` and the deployment yaml are still relevant. They seem unused.